apps/vr-tests-react-components/src/stories/TagPicker.stories.tsx

```diff
@@ -0,0 +1,425 @@
+import * as React from 'react';
+import { ComponentMeta } from '@storybook/react';
+import {
+  TagPicker,
+  TagPickerControl,
+  TagPickerGroup,
+  TagPickerInput,
+  TagPickerList,
+  TagPickerOption,
+  TagPickerOptionGroup,
+  TagPickerProps,
+} from '@fluentui/react-tag-picker-preview';
+import { Tag } from '@fluentui/react-tags';
+import { Avatar } from '@fluentui/react-avatar';
+import { DARK_MODE, HIGH_CONTRAST, RTL, getStoryVariant } from '../utilities/getStoryVariant';
+import { makeStyles, shorthands } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
+import { Steps, StoryWright } from 'storywright';
+import { ArrowDownFilled } from '@fluentui/react-icons';
+import { Button } from '@fluentui/react-button';
+
+export default {
+  title: 'TagPicker',
+} as ComponentMeta<typeof TagPicker>;
+
+const options = [
+  'John Doe',
+  'Jane Doe',
+  'Max Mustermann',
+  'Erika Mustermann',
+  'Pierre Dupont',
+  'Amelie Dupont',
+  'Mario Rossi',
+  'Maria Rossi',
+];
+
+export const Default = (props: Pick<TagPickerProps, 'appearance' | 'size'>) => {
+  const [selectedOptions, setSelectedOptions] = React.useState<string[]>([]);
+  const onOptionSelect: TagPickerProps['onOptionSelect'] = (e, data) => {
+    setSelectedOptions(data.selectedOptions);
+  };
+
+  return (
+    <div style={{ maxWidth: 400 }}>
```

Comment:
nit: let's add some padding, making it easier to see the picker's border in dark/hc background